### ISSUE TITLE
consider to include alibaba spring-context-support source 

### ISSUE THREAD

**Issue by beiwei30** [apache/incubator-dubbo#2701]
alibaba spring-context-support is purely some utilities on spring, pls. consider to directly include its source into the trunk.

```xml
<dependency>
  <groupId>com.alibaba.spring</groupId>
  <artifactId>spring-context-support</artifactId>
  <version>1.0.2</version>
</dependency>
```

**Comment by zonghaishang** [comment]
+1

**Comment by beiwei30** [comment]
this is really annoying, people have to remember to include this particular dependency every time when they use spring, in my opinion, it hurts user experience.

**Comment by beiwei30** [comment]
it's fixed in 78accf78
